lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -110,7 +111,7 @@ def display_peek(self, dataset):
 
 class PlyAscii(Ply, data.Text):
 
-    file_ext = "plyascii"
+    file_ext = "ply"
```

Comment:
Still many tools don't distinguish between the extensions. Havn't seen any tool which saves a file in xxx.vtkascii. A correct file extension is mandatory for https://github.com/ThomasWollmann/docker-paraviewweb.